Documentation/HowToGuides/AddingAnInteractor/README.md

```diff
@@ -0,0 +1,68 @@
+
+### Step 2
+
+Select the `Interactions.Interactor` prefab in the Unity Hierarchy under the `LeftControllerAlias` GameObject and change the `Interactor Facade` component to configure the base functionality of the Interactor.
+
+We need to specify an action that initiates the Interactor’s Grab notification. The Grab Action parameter on the Interactor Facade allows us to provide a Boolean Action to initiate the Grab function of the Interactor.
+
+For this example we will set up the pointer to use the simulator controls utilizing the mouse.
+
+Expand `CameraRigs.SpatialSimulator -> Input -> Control Objects Input -> Right Controller Input -> Buttons`.
+
+`Button One` is the left mouse button and we'll use this to activate the grab action on our left interactor so drag and drop the `Button One` GameObject into the `Grab Action` parameter on the Interactor Facade component on the `Interactions.Interactor` prefab nested inside `LeftControllerAlias` Gameobject
+
+![Drag And Drop Button One Into Grab Activation Parameter](assets/images/DragAndDropButtonOneIntoGrabActivationParameter.png)
+
+### Step 3
+
+When an Interactor grabs an interactable object, it is usually required that the velocity that the Interactor is moving at can be applied to the interactable object. For example, if we were to pick up an interactable object and throw it, then we would want the velocity that we’re moving our virtual controller at to be applied to the interactable object so we can control the power of the throw.
+
+The `Velocity Tracker` parameter on the `Interactor Facade` component allows us to specify a velocity tracking component that will provide the relevant velocity data. Fortunately, our TrackedAlias prefab already comes with the relevant `Velocity Tracker` components on the relevant aliases that can move freely in the virtual world such as the Headset, Left Controller and Right Controller.
+
+Drag and drop the `CameraRigs.TrackedAlias -> Aliases -> LeftControllerAlias` GameObject into the `Velocity Tracker` parameter on the Interactor Facade component for the `Interactions.Interactor` prefab nested under the `LeftControllerAlias` GameObject to set the interactor to track velocities from our Left Controller.
+
+![Drag And Drop Left Controller Alias Into Velocity Tracker Parameter](assets/images/DragAndDropLeftControllerAliasIntoVelocityTrackerParameter.png)
+
+> Note: Be sure to do the same with the RightControllerAlias GameObject and the RightControllerAlias -> Interactor prefab.
+
+### Done
+
+We now have an Interactor prefab attached to our TrackedAlias -> LeftControllerAlias and our TrackedAlias -> RightControllerAlias and Interactors will follow the Alias GameObjects as if they are children of the GameObjects in the Unity Hierarchy. At the moment, these Interactors won’t do anything special in our scene because there is nothing to interact with.
```

Comment:
full name for `Interactions.Interactor` prefab

backtick and full name for:
`CameraRigs.TrackedAlias -> Aliases -> LeftControllerAlias` 
and
`CameraRigs.TrackedAlias -> Aliases -> RighttControllerAlias`